src/main.rs

```diff
@@ -247,8 +247,8 @@ async fn run(error_sender: Sender<anyhow::Error>) -> Result<()> {
 	let last_full_node_ws = data::get_last_full_node_ws_from_db(db.clone())?;
 
 	let version = rpc::Version {
-		version: "1.6.1".to_string(),
-		spec_version: 10,
+		version: "1.6.1-7fcca43d5f5".to_string(),
```

Comment:
I don't think that hash is necessary.

Reply:
I agree. This might mess up the version output.